### PR TITLE
Add new proposals related to Kubernetes Dashboard

### PR DESCRIPTION
**Pull request by maciaszczykm**
Still few things to clarify.

/cc @floreks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ Please visit the [Kubernetes GSoC page](https://git.k8s.io/community/mentoring/g
 -	Mentor(s): Andy Zhang (@andyzhangx)
 -	Issue: https://github.com/kubernetes/org/issues/344
 
+### Kubernetes Dashboard
```
Comment (nikhita) [inline README.md]: @maciaszczykm This will go as a project idea for Kubernetes directly, so this isn't needed :)

Comment (maciaszczykm) [reply README.md]: Okay, will change 👍

Comment (maciaszczykm) [reply README.md]: Done, thanks :) Still thinking how to describe the second idea...

**Comment by nikhita** [comment]
@maciaszczykm If you want to add the second idea in another PR and merge this one, that's also ok :)

**Comment by floreks** [comment]
LGTM :slightly_smiling_face:

**Comment by maciaszczykm** [comment]
@nikhita WDYT?

**Comment by nikhita** [comment]
Looks great! If you squash the commits to a single commit, I'll merge this. :smile:

**Comment by maciaszczykm** [comment]
Done!

**Pull request merged**